LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDevice.cs

```diff
@@ -292,7 +292,7 @@ public virtual async Task<bool> InitializeAsync(NetworkServerConfiguration confi
                 // OTAA
                 try
                 {
-                    AppKey = desiredTwin.ReadRequiredString(TwinProperty.AppKey);
+                    AppKey = desiredTwin.TryRead(TwinProperty.AppKey, out AppKey someAppKey) ? someAppKey : null;
```

Comment:
I think this is wrong. We need to make sure we have an AppKey and throw if there is none for OTAA. Otherwise the device could never join.

Reply:
Done in 6f3c2197acaa8a6f83a14fe9e4b34e7056b33bf4